05_run_ocp.sh

```diff
@@ -88,8 +88,8 @@ done
 sudo virsh domifaddr ${CLUSTER_NAME}-bootstrap
 
 IP=$(sudo virsh domifaddr ostest-bootstrap | grep 122 | awk '{print $4}' | grep -o '^[^/]*')
-# TODO point to libvirt's DNS instead once correct hostname is set
-echo "address=/${CLUSTER_NAME}-api.${BASE_DOMAIN}/${IP}" | sudo tee /etc/NetworkManager/dnsmasq.d/openshift.conf
+echo "addn-hosts=/etc/hosts.openshift" | sudo tee /etc/NetworkManager/dnsmasq.d/openshift.conf
+echo "${IP} ${CLUSTER_NAME}-api.${BASE_DOMAIN}" | sudo tee /etc/hosts.openshift
```

Comment:
In https://github.com/metalkube/dev-scripts/pull/38 I'm going to move the api to be a VIP so that we can safely move it from the bootstrap to the cluster masters. Until then. I'm more than fine with going this way.

Maybe let's add a NOTE like:

```
# NOTE: This hardcodes CLUSTER_NAME-api.BASE_DOMAIN to the bootstrap node.
# TODO: Point instead to the DNS VIP when we have that. E.g.: server=/BASE_DOMAIN/DNS_VIP

Reply:
Added the note.

DNSMasq hacks are fairly useful at this point, but eventually we should move to libvirt on this if possible